app/services/analytics.rb

```diff
@@ -123,11 +123,8 @@ def browser_attributes
 
   # rubocop:disable Layout/LineLength
   ACCOUNT_RESET_VISIT = 'Account deletion and reset visited'
-  BROKEN_PERSONAL_KEY_REGENERATED = 'Broken Personal Key: Regenerated'
   DOC_AUTH = 'Doc Auth' # visited or submitted is appended
-  DOC_AUTH_ASYNC = 'Doc Auth Async'
   DOC_AUTH_WARNING = 'Doc Auth Warning'
```

Comment:
Were we expecting to migrate all these?

Reply:
yes, I forgot to remove the rest, done in ca57cc9f53112214fdd860dc6887a56e1f33c7fd